app-rails/config/environments/development.rb

```diff
@@ -66,6 +66,9 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
+  # Allow web_console to render when triggered from the rails app running locally in a docker container.
+  config.web_console.permissions = ["192.168.0.0/16", "172.16.0.0/16", "10.0.0.0/8"]
+
```

Comment:
Did you find that this was necessary? I don't think I needed to do this to access the web console.

Reply:
Got it. Thank you!

Reply:
I have confirmed your fix is necessary. I wonder if I made the fix in some branch and then totally forgot about it and lost it. Thanks for explaining.